utils/api/configurations.js

```diff
@@ -106,13 +106,15 @@ export const configureStatus = (status) => {
 export const configureMessages = (data) => {
   // NOTE(alishaevn): doing some basic filtering here until we come to a resolution on
   // https://github.com/assaydepot/scientist_api_v2/issues/167
-  const filteredMessages = data.filter(d => d.user_ref && d.body)
+  const filteredMessages = data.filter(d => d.user_ref && d.body && d.created_at && d.attachments)
 
   return filteredMessages.map(note => ({
     avatar: note.user_ref.image,
     body: note.body,
     id: note.id,
     name: `${note.user_ref.first_name} ${note.user_ref.last_name}`,
+    dateAndTime: note.created_at,
```

Comment:
this date needs to be normalized in some way doesn't it?

Reply:
I see that you're doing some stuff with the date in the library. I'm inclined to move `formatBytes` and `timeSince` into the webstore instead though since the library should be presentational as much as possible. only caring about the display, not manipulating things.

and we're going to have to configure `attachments` too.

Reply:
it gets turned into timeSince in the component library here: https://github.com/scientist-softserv/webstore-component-library/pull/103/files#diff-1a35a2cedfe808a2b9278301ab85c1674c26fef0890bcedac0382f2dcd7cc9d0

Reply:
oh weird i just saw your second comment load after posting that. ok i will move them here and out of the webstore

Reply:
but what about convertToBase64? why should that stay in the component library